api/conf/conf.json

```diff
@@ -10,6 +10,17 @@
     "dag-lib-path": "",
     "etcd": {
       "endpoints": "127.0.0.1:2379"
+    },
+    "log": {
+      "error_log": {
+        "level": "warn",
+        "file_path": ""
```

Comment:
what means empty string? It makes me confused

Reply:
@membphis  empty string means don't save to a file.

Reply:
We’d better update our doc to let users know what this means.

I would prefer using a better way to resolve this confusion, instead of using docs. @membphis , do you have any experiences?

Reply:
Our goal is to let code tell us what is it and what it does :)

Reply:
> empty string means don't save to a file.

I do not think that is a good design.

How about this way?

```sh
# case 1:
file_path = "logs/error.log"  # write error log to file `./logs/error.log` in current work dir

# case 2:
file_path = "/tmp/logs/error.log"  # write error log to file `/tmp/logs/error.log`

# case 3:
file_path = "/dev/stdout" # print the error log to stand output device

# case 4
file_path = "/dev/stderr" # print the error log to error output device
```

For an empty string, it should be an invalid value, if the user uses this value, we'll throw an error message. 

@nic-chen

Reply:
> We’d better update our doc to let users know what this means.

we should choose a good design first. 
the user knows how to use it without any explanation is the best way.

Reply:
`case 1` + 1